bedrock-devnet/devnet/__init__.py

```diff
@@ -43,13 +46,20 @@ def main():
       deploy_config_dir=pjoin(contracts_bedrock_dir, 'deploy-config'),
       op_node_dir=op_node_dir,
       ops_bedrock_dir=ops_bedrock_dir,
+      ops_chain_ops=ops_chain_ops,
+      sdk_dir=sdk_dir,
       genesis_l1_path=pjoin(devnet_dir, 'genesis-l1.json'),
       genesis_l2_path=pjoin(devnet_dir, 'genesis-l2.json'),
       addresses_json_path=pjoin(devnet_dir, 'addresses.json'),
       sdk_addresses_json_path=pjoin(devnet_dir, 'sdk-addresses.json'),
       rollup_config_path=pjoin(devnet_dir, 'rollup.json')
     )
 
+    if args.test:
+      log.info('Testing deployed devnet')
+      devnet_test(paths)
```

Comment:
The idea here is that this test command is independent of bringing up the stack?

Reply:
Correct.  To run the test manually, you'd do something like:

```
make devnet-up-deploy
make devnet-test
```

I'm not sure how everyone is using the `devnet` targets, but it seems plausible that some users are running other loads against the devnet, so I was wary of including the test execution with bringing the devnet up or down.

Reply:
you would have `devnet-up-deploy` be a pre-requisite for `devnet-test` if you wanted `devnet-test` to automatically bring it up. The only potential issue is that it might always restart the devnet.

Reply:
Right.  Conceivably, we could have multiple targets if we wanted to allow both, say `devnet-test-only` and `devnet-test` which required the `devnet-up-deploy` followed by `devnet-test-only`.  If there's some consensus, I'm happy to modify.